eva_submission/eload_submission.py

```diff
@@ -172,7 +172,6 @@ def replace_values_in_metadata(self, taxid=None, reference_accession=None):
 
     def detect_all(self):
         self.detect_submitted_metadata()
-        self.detect_submitted_vcf()
```

Comment:
Right, this is happening elsewhere now... except maybe actually checking the vcf files exist in the file system, should we still be doing that?

Also, does this mean we can get rid of the `detect_submitted_vcf` method altogether?

Reply:
That check will be moved to the preparation as part of the [EVA-2459](https://www.ebi.ac.uk/panda/jira/browse/EVA-2459). I though it would be a good idea to keep the `detect_submitted_vcf` until that ticket is done as it can be useful

Reply:
It makes sense to keep it here, @andresfsilva initially did something sensible by just commenting it out but I got confused, my bad.  Either way I'll add it back / modify it in #45 so it shouldn't matter what we do with it here.